api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -155,6 +155,9 @@ public DbDemographicSurvey apply(DemographicSurvey demographicSurvey) {
               if (demographicSurvey.getDisability() != null)
                 result.setDisabilityEnum(
                     demographicSurvey.getDisability() ? Disability.TRUE : Disability.FALSE);
+              if (demographicSurvey.getGender() != null) {
+                result.setGenderEnum(demographicSurvey.getGender());
```

Comment:
I think we should consider making it a bad request if NONE or PREFER_NO_ANSWER aren't by themselves. The UI should probably enforce that as well.

We should definitely allow combining the others though.

Reply:
Since that logic is not just limited to gender we can have another PR/story to take care of it.